rust/tests/namespace.ts

```diff
@@ -647,19 +651,21 @@ describe("namespace", () => {
     const joinNsAccounts: nsIx.JoinNamespaceAccounts = {
       namespaceMint: nsMint,
       artifact: itemClass[0],
+      raindropsProgram: pids.ITEM_ID,
     };
 
     const joinNsResult = await namespaceProgram.joinNamespace(joinNsAccounts);
     console.log("joinNsTxSig: %s", joinNsResult.txid);
 
     const [namespace, _namespaceBump] = await pdas.getNamespacePDA(nsMint);
-    var nsData = await namespaceProgram.fetchNamespace(namespace);
+    let nsData = await namespaceProgram.fetchNamespace(namespace);
```

Comment:
https://github.com/raindrops-protocol/raindrops/pull/28/files#r945335531

Reply:
I think this is fine how it is personally

Reply:
Why? If you're not going to use the type system in the way suggested, why even change it from using `var`?